server/build.gradle

```diff
@@ -110,19 +103,15 @@ configurations {
   }
 
   sharedTestOutput {
-    extendsFrom testRuntime
+    extendsFrom testRuntimeClasspath
     transitive = true
   }
 
-  compile.exclude(group: 'org.springframework', module: 'spring-support')
-  runtime.exclude(group: 'org.springframework', module: 'spring-support')
-
-  // this dependency is a "provided" dependency
-  compile.exclude(group: 'ch.qos.logback', module: 'logback-classic')
-  runtime.exclude(group: 'ch.qos.logback', module: 'logback-classic')
-  apiBase.exclude(group: 'ch.qos.logback', module: 'logback-classic')
-  api.exclude(group: 'ch.qos.logback', module: 'logback-classic')
-  spark.exclude(group: 'ch.qos.logback', module: 'logback-classic')
```

Comment:
All of this exclusion logic is handled during packaging in the `war` task. See the lines pertaining to configurations "arithmetic" when building the `classpath`.